boilerplates/custom-post-type.php

```diff
@@ -27,11 +26,11 @@ function __construct() {
 	 */
 	function action_init() {
 		$labels = array(
-			'name'               => esc_html_x( self::POST_TYPE_NAME, 'post type general name', 'braces' ),
-			'singular_name'      => esc_html_x( self::POST_TYPE_SINGULAR, 'post type singular name', 'braces' ),
-			'menu_name'          => esc_html_x( self::POST_TYPE_NAME, 'admin menu', 'braces' ),
-			'name_admin_bar'     => esc_html_x( self::POST_TYPE_SINGULAR, 'add new on admin bar', 'braces' ),
-			'add_new'            => esc_html_x( 'Add New '. self::POST_TYPE_SINGULAR, 'post type', 'braces' ),
+			'name'               => _x( self::POST_TYPE_NAME, 'Post Type General Name', 'braces' ),
+			'singular_name'      => _x( self::POST_TYPE_SINGULAR, 'Post Type Singular Name', 'braces' ),
+			'menu_name'          => _x( self::POST_TYPE_NAME, 'Admin Menu', 'braces' ),
+			'name_admin_bar'     => _x( self::POST_TYPE_SINGULAR, 'Add New on Admin Bar', 'braces' ),
+			'add_new'            => _x( 'Add New '. self::POST_TYPE_SINGULAR, 'Post Type', 'braces' ),
```

Comment:
This isn't actual legal – you can't use anything other than purely literal strings in translation functions, even though this is a constant.

Try `sprintf( _x( 'Add New %s', 'Post Type', 'braces' ), self::POST_TYPE_SINGULAR )` instead.